### PR TITLE
Add exploration info + Launch new E&A page

### PR DESCRIPTION
**Pull request by aboydnw**
Adding info to prep for releasing the new exploration and analysis feature to the Earthdata instance
Relates to #391

### CODE REVIEW

File: datasets/cmip6-tas.data.mdx [hunk 1]
```diff
@@ -13,6 +13,18 @@ taxonomy:
...
+    * Format: [kerchunk (metadata)](https://fsspec.github.io/kerchunk/) for netCDF4
+    * Spatial Coverage: 180° W to 180° E, 60° S to 90° N
+    * Temporal: 1950-01-01 to 1951-12-31
+      * _As noted below, this dataset is a subset all available data. The full dataset includes data from 1950 to 2100._
+    * Data Resolution:
+      * Latitude Resolution: 0.25 degrees (25 km)
+      * Longitude Resolution: 0.25 degrees (25 km)
+      * Temporal Resolution: daily
+    
+    Source: [https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-gddp-cmip6](https://www.nccs.nasa.gov/services/data-collections/land-based-products/nex-gddp-cmip6)
```
Comment (sandrahoang686) [inline datasets/cmip6-tas.data.mdx:27]: I think the issue might be here 🤔 I dont think you can add a source at this level like so. This is fine in the Block area though. I'd try removing this to see if the error goes away

Comment (sandrahoang686) [reply datasets/cmip6-tas.data.mdx]: _not the issue_

Comment (aboydnw) [reply datasets/cmip6-tas.data.mdx]: Good to know, I just removed the source reference from this section. Thanks for catching!

**Review by hanbyul-here — APPROVED** [review]

**Comment by aboydnw** [comment]
This is ready for merge per slack convo with Brian

**Pull request merged**